timesheet_edit_only_today/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Statement seems to have no effect](https://app.codacy.com/manual/numigi/odoo-timesheet-addons/pullRequest?prid=4979202)